website/src/components/genspectrum/GsWastewaterMutationsOverTime.astro

```diff
@@ -7,15 +7,19 @@ interface Props {
     title: string;
     lapisFilter: LapisFilter;
     sequenceType: SequenceType;
+    info?: string;
 }
 
-const { title, lapisFilter, sequenceType } = Astro.props;
+const { title, lapisFilter, sequenceType, info } = Astro.props;
 ---
 
 <ComponentWrapper title={title}>
     <gs-wastewater-mutations-over-time
         lapisFilter={JSON.stringify(lapisFilter)}
         sequenceType={sequenceType}
         width='100%'
-        height='100%'></gs-wastewater-mutations-over-time>
+        height='100%'
+    >
+        <span slot='infoText'><Fragment set:html={info} /></span>
```

Comment:
Why do we need that `Fragment` here?

Reply:
Currently, the info text is provided as a string with HTML (see [here](https://github.com/GenSpectrum/dashboards/pull/497/files#diff-42f81b60a3c67d368026866e4b38927feeaf1f3b72f29bcb60a80729048f24f7R38)), this way, we need a Fragment, don't we? Is there a better way?

Reply:
Ah I see... Why don't we use an Astro slot here?
```suggestion
        <span slot='infoText'><slot name="infoText" /></span>
```

And then, where the component is called, we can use it as 

```
<gs-wastewater...>
  <span slot="infoText">text... <a>important link</a></span>
</gs-wastewater...>
``

Reply:
this didn't work btw - and I decided as we are under time constraints to merge without